iocage_lib/ioc_fstab.py

```diff
@@ -228,8 +231,11 @@ def __validate_fstab__(self, fstab, mode='single', actions=None):
 
             if not source.is_dir():
                 if fstype == 'nullfs':
-                    if pathlib.Path('/mnt/iocage') in source.parents or \
-                            pathlib.Path('/iocage') in source.parents:
+                    if any(
+                        pathlib.Path(i) in source.parents for i in (
```

Comment:
What about altroots that aren't any of these. Or different mountpoints?

Reply:
All of these scenarios are cases where we are correcting past iocage behavior, do you have any other examples for the scenarios you mention which we explicitly created ?

Reply:
No just me not reading the diff closely enough 😂

Reply:
Hahaha :D No worries, happens to all of us ;)